README.md

```diff
@@ -121,6 +121,12 @@ int main(int argc, char* argv[]) {
   }
 
   auto reader = client->ReadObject(bucket_name, "quickstart.txt");
+  if (reader.bad()) {
```

Comment:
I think this should be `if (!reader)` instead of `if (!reader.bad())`. What do you think?

This quickstart is copied from the storage readme (and consequently from quickstart.cc). I think you should update all three places for consistency.

Reply:
Actually, you should update `quickstart.cc` then run `ci/generate-markdown/generate-storage-readme.sh`

Reply:
yes you are right.
i modified it.